iosApp/iosApp/ComponentViews/LocationAuthButton.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  LocationAuthButton.swift
+//  iosApp
+//
+//  Created by Jack Curtis on 10/28/24.
+//  Copyright © 2024 MBTA. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct LocationAuthButtonStyle: ButtonStyle {
+    func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .padding()
+            .background(Color.key)
+            .foregroundStyle(.white)
+            .clipShape(Capsule())
+    }
+}
+
+struct LocationAuthLabelStyle: LabelStyle {
+    func makeBody(configuration: Configuration) -> some View {
+        HStack(alignment: .center, spacing: 8) {
+            configuration.title
+            configuration.icon
+        }
+        .fontWeight(.bold)
+    }
+}
+
+struct LocationAuthButton: View {
+    @EnvironmentObject var locationDataManager: LocationDataManager
+
+    @Binding var showingAlert: Bool
+
+    var body: some View {
+        switch locationDataManager.authorizationStatus {
+        case .notDetermined, .denied, .restricted:
+            Button(action: {
+                showingAlert = true
+            }, label: {
+                Label(title: {
+                    Text("Location Services is off")
+                }, icon: {
+                    Image(.faChevronRight)
+                        .resizable()
+                        .scaleEffect(0.6)
+                        .frame(width: 20, height: 20, alignment: .center)
+                        .backgroundStyle(.white)
+                        .background(in: .circle)
+                        .foregroundColor(Color.key)
+                })
+                .labelStyle(LocationAuthLabelStyle())
+            })
+            .buttonStyle(LocationAuthButtonStyle())
+            .accessibilityIdentifier("locationServicesButton")
+            .alert(
+                "Maps work best with Location Services turned on",
+                isPresented: $showingAlert,
+                actions: {
+                    Button("Turn On in Settings") {
+                        showingAlert = false
+                        if let url = URL(string: UIApplication.openSettingsURLString) {
+                            UIApplication.shared.open(url)
+                        }
+                    }
+                    Button("Keep Location Services off") {
```

Comment:
Could we also capitalize "Off" for consistency?

Reply:
I believe we use sentence case for this copy, it's just that Location Services is a proper noun so it gets capitalized

Reply:
Oh but it is capitalized in the spec. I'll correct it then.